tests/app/main/test_frameworks.py

```diff
@@ -1679,13 +1706,13 @@ def test_confidence_banner_not_on_page_if_sections_incomplete(
         self.login()
         res = self.client.get("/suppliers/frameworks/g-cloud-8")
         assert res.status_code == 200
-        body = res.get_data(as_text=True)
-        assert self.expected not in body
 
-        # This confirms that the test is working correctly - i.e. the banner is not showing because a specific section
-        # is causing the application to be incomplete.
-        for text in check_text_in_doc:
-            assert text in body
+        document = html.fromstring(res.get_data(as_text=True))
+
+        # Alert banner should not be shown
+        alert_banner = document.xpath('//div[@class="dm-alert dm-alert--success"]')
+        assert len(alert_banner) == 0
+        assert 'Your application is complete, and will be automatically submitted.' not in res.get_data(as_text=True)
```

Comment:
Suggest amend: Your application is complete and will be submitted automatically.